src/main/java/com/vmtapp/enterprise/dto/Ticket.java

```diff
@@ -13,7 +13,7 @@
 class Ticket {
     @Id
     @GeneratedValue(strategy= GenerationType.AUTO)
-    public int id;
+    public int TicketId;
```

Comment:
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
"variable names short and describe what it stores"

Reply:
The concept of "Ticket" is a given from the context: the class itself.  So, no need to repeat the name inside the attribute.  None of the other attributes are prefixed with ticket.